db/migrate/20170707150520_update_container_template_types.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateContainerTemplateTypes < ActiveRecord::Migration[5.0]
+  class ContainerTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Updating type column for ContainerTemplate") do
+      ContainerTemplate.update_all(:type => "ManageIQ::Providers::ContainerManager::ContainerTemplate")
```

Comment:
should it be `ManageIQ::Providers::OpenShift::ContainerManager::ContainerTemplate` or derive the class name with actual provider name from its container manager?